karapace/schema_reader.py

```diff
@@ -254,105 +266,122 @@ def handle_messages(self):
                 if self.ready and add_offsets:
                     self.offset_watcher.offset_seen(self.offset)
 
-    def handle_msg(self, key: dict, value: dict):
-        if key["keytype"] == "CONFIG":
-            if "subject" in key and key["subject"] is not None:
-                if not value:
-                    self.log.info("Deleting compatibility config completely for subject: %r", key["subject"])
-                    self.subjects[key["subject"]].pop("compatibility", None)
```

Comment:
This was another bug, this would throw if the first message in the topic was delete the config compatibility. For Karapace only topics this cant happen, because the delete endpoint is missing https://github.com/aiven/karapace/issues/380. This would only be an issue for a CSR -> Karapace migration, where the topic config was delete and it happened to be the first message in the topic.